mistletoe/block_token.py

```diff
@@ -359,6 +359,11 @@ def __init__(self, lines):
         self.language = ''
         self.children = (span_token.RawText(''.join(lines).strip('\n')+'\n'),)
 
+    @property
+    def content(self):
+        """Return the code block content."""
```

Comment:
OK, just a slight detail for consistency:

```suggestion
        """Returns the code block content."""
```

DITTO below.

Reply:
Ok, fixed